controllers/kubeadmbootstrapconfig_controller.go

```diff
@@ -93,6 +98,30 @@ func (r *KubeadmBootstrapConfigReconciler) Reconcile(req ctrl.Request) (ctrl.Res
 		log.Error(err, "failed to get cluster")
 		return ctrl.Result{}, errors.WithStack(err)
 	}
+
+	// If the machine is not Ready, requeue until it's ready
```

Comment:
This needs to be removed - the machine controller determines bootstrapReady based on the value in the bootstrap resource's `status.ready` field (which, in this case, is this code here!).

Reply:
ahh ok, so once the bootstrap config becomes ready then the machine controller will update the machine.status.BootstrapReady to true. yes, this should be removed in that case.